inlong-dashboard/src/plugins/sources/common/SourceDefaultInfo.ts

```diff
@@ -146,10 +147,16 @@ export class SourceDefaultInfo implements DataWithBackend, RenderRow, RenderList
   readonly modifier: string;
 
   parse(data) {
+    if (data.resetTime !== undefined) {
+      data.resetTime = dayjs(data.resetTime).format('YYYY-MM-DD HH:mm:ss');
```

Comment:
Is resetTime a parameter that all Source plugins have? If not, do not add it to default.